flink-connectors/flink-connector-files/pom.xml

```diff
@@ -34,6 +34,13 @@ under the License.
 
 	<packaging>jar</packaging>
 
+	<properties>
+		<surefire.module.config><!--
+			chill ArraysAsListSerializer
```

Comment:
We need to add this because `ArrayAsListSerializer` is used within `com.twitter:chill-java:jar:0.7.6` which is a test dependency within `flink-java`? Therefore, any module that depends on `flink-java` in its test code has to add this part to the surefire configuration?!